docs/guides/implementation-guides/how-to-implement-dependency-injection.md

```diff
@@ -5,7 +5,7 @@ title: How To Implement Dependency Injection
 
 # How To Implement Dependency Injection in Avalonia
 
-[Dependency injection (DI)](https://en.wikipedia.org/wiki/Dependency_injection) allows developers to write cleaner, more modular, and testable code. It is especially useful when you are developing multi-platform applications that need specific service implementations for each platform.
+[Dependency injection (DI)](https://en.wikipedia.org/wiki/Dependency_injection) allows developers to write cleaner, more modular, and testable code.It accomplishes this by creating discrete services that are passed around/created as needed.q It is especially useful when you are developing multi-platform applications that need specific service implementations for each platform.
```

Comment:
You are missing a space before "it accomplishes". You also have a errant "q" before "It is especially useful"